pkg/apis/azure/storage/v1alpha1/account_test.go

```diff
@@ -49,7 +49,7 @@ func Test_newCustomDomain(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got := newCustomDomain(tt.args)
-			if diff := deep.Equal(got, tt.want); diff != nil {
+			if diff := cmp.Diff(got, tt.want); diff != "" {
 				t.Errorf("newCustomDomain() = %v, want %v\n%s", got, tt.want, diff)
```

Comment:
I like the paradigm from the above, with detailed `diff` there is no need to display either `got` nor `tt.want`.
```go
t.Errorf("newCustomDomain() %s", diff)
```